tests/commands_tests/test_setupproject.py

```diff
@@ -85,7 +85,7 @@ def mocked_resolve_core_path(core_path):
         new_project = {
             "type": "Project",
             "id": 1234,
-            "code": "new_project_1234",
+            "name": "new_project_1234",
```

Comment:
Bad test data. Using wrong field to name entity. More on why this change was introduced bellow.